src/common/EIP712/TypedRequestData.ts

```diff
@@ -5,6 +5,8 @@ import { EIP712Domain, EIP712TypedData, EIP712TypeProperty, EIP712Types, TypedDa
 import { bufferToHex } from 'ethereumjs-util'
 import { PrefixedHexString } from 'ethereumjs-tx'
 
+require('source-map-support').install({ errorFormatterForce: true })
```

Comment:
Does it work?

Reply:
don't remember... added it in several places, and at least in some it helped. we should have an "import" done everywhere (so its "import", and not "require"), and if we find a better fix, we'll put it there, in one place.